libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFNetworkPlugin/SFNetworkPlugin.m

```diff
@@ -125,14 +125,15 @@ - (void) pgSendRequest:(CDVInvokedUrlCommand *) command
     [restApiInstance sendRequest:request
                                       failureBlock:^(id response, NSError *e, NSURLResponse *rawResponse) {
                                           __strong typeof(self) strongSelf = weakSelf;
-                                          NSMutableDictionary *errorDictionary = [NSMutableDictionary new];
+                                          NSMutableDictionary *responseDictionary = [[rawResponse asDictionary] mutableCopy];
                                           if ([response isKindOfClass:[NSDictionary class]] || [response isKindOfClass:[NSArray class]]) {
-                                              errorDictionary[@"response"] = response;
+                                              responseDictionary[@"body"] = response;
```

Comment:
What about headers and statusCode?

Reply:
Never mind I just reviewed that code https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/dev/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/NSURLResponse%2BSFAdditions.m ;-)